<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DelProject</class>
 <widget class="QWidget" name="DelProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>所有工程</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>所有工程：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="listView"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>描述：</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="currentButton">
       <property name="text">
        <string>设置为当前工程</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delButton">
       <property name="text">
        <string>删除此工程</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>DelProject</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
